src/commands/reserve.ts

```diff
@@ -66,16 +73,33 @@ export default class Reserve extends CommandWithCatch {
       return;
     }
 
-    const availreq = makeAvailabilityRequest(flags.start, flags.duration, flags.lattice);
+    if (flags.list) {
+      const availreq = makeAvailabilityRequest(flags.start, flags.duration, flags.lattice);
+      const availabilities = await GET.availability(availreq);
+      this.log(serializeAvailabilities(availabilities, flags.format as SerializeFormat));
+      return;
+    }
+    if (flags.confirm) {
```

Comment:
I think it makes sense to have a separate flag for booking directly without having to select from a menu that's separate from the notion of `--confirm` overriding yes/no confirmation steps.